pkgs/by-name/libgnunetchat/package.nix

```diff
@@ -18,7 +18,7 @@
 # From https://github.com/ngi-nix/gnunet-messenger-cli/blob/main/flake.nix
 stdenv.mkDerivation (finalAttrs: {
   name = "libgnunetchat";
-  version = "0.5.0-unstable-2025-01-07";
+  version = "0.5.3";
 
   src = fetchgit {
```

Comment:
Changing the version is not enough here. We also need to update the hashes as well, replacing the commit rev with the tag since we're targeting a stable release.